libs/libpcache/src/config/config.cpp

```diff
@@ -268,45 +217,105 @@ unordered_map<string,vector<Fault*>> Config::load_config (string filename) {
 
                 if (injection.contains("parts") && injection.contains("parts_bytes")) throw std::runtime_error("Keys 'parts' and 'key_parts' for some injection of type \"torn-op\" are exclusive in the configuration file. Please define at most one of them.");     
 
-                cache::config::SplitWriteF * fault = NULL;
+                faults::SplitWriteF * fault = NULL;
 
                 //A split write fault either contains the parameter "parts" or the "pats_bytes"
                 if (injection.contains("parts")) {
                     int parts = toml::find<int>(injection,"parts");
                     
-                    if (parts <= 0) throw std::runtime_error("Key 'parts' for some injection of type \"torn-op\" has an invalid value in the configuration file. It should be greater than 0.");
+                    vector<string> errors = faults::SplitWriteF::check_with_errors(occurrence,persist,parts,std::nullopt);
+                    if (errors.size() > 0) {
+                        string error_msg = "The following errors were found in the configuration file for the injection of type \"torn-op\": ";
```

Comment:
You should report a configuration file error, but lazyfs should not crash or exit, since these errors do not prevent its normal execution.